apps/web/src/routes/GroupSettings/RolesSettingsTab/RolesSettingsTab.js

```diff
@@ -262,7 +264,7 @@ function RoleRow ({
   )
 }
 
-function AddMemberToRole ({
+export function AddMemberToRole ({
```

Comment:
ideally we only export one component per file, so that the hot module reloading works. so we should move these into their own files to test them